File: ROS_Projects/src/my_package/scripts/publisher.py
#!/usr/bin/env python
from std_msgs.msg import String
import rospy
if __name__ == '__main__':
    rospy.init_node('publisher_node')
    pub = rospy.Publisher('/random_topic',String,queue_size=10)
    rate = rospy.Rate(2)
    while not rospy.is_shutdown():
        msg = String()
        msg.data = 'Hello !'
        pub.publish(msg)
        rate.sleep()
    rospy.loginfo('Node killed. Program terminating ...')
    

File: Asimov_Workspace/src/assignment/src/publisher.cpp
#include<ros/ros.h>
#include<std_msgs/String.h>
int main(int argc, char *argv[])
{
    ros::init(argc, argv, "publisher");
    ros::NodeHandle nh;
    ros::Publisher pub1 = nh.advertise<std_msgs::String>("/text1", 1000);
    ros::Publisher pub2 = nh.advertise<std_msgs::String>("/text2", 1000);
    ros::Rate rate(1);
    std_msgs::String msg;
    msg.data = "hello world !";

    while(ros::ok())
    {
        pub1.publish(msg);
        pub2.publish(msg);
        rate.sleep();
    }

    return(0);
}